### PR TITLE
Add support for audit logging

### PR DESCRIPTION
**Pull request by palourde**

### CODE REVIEW

File: uchiwa/config/config.go [hunk 1]
```diff
@@ -160,6 +167,13 @@ func (c *Config) initUchiwa() {
 		c.Uchiwa.Users = append(c.Uchiwa.Users, auth.User{Username: c.Uchiwa.User, Password: c.Uchiwa.Pass, FullName: c.Uchiwa.User})
 	}
 
+	// audit
+	if c.Uchiwa.Audit.Level != "verbose" && c.Uchiwa.Audit.Level != "disabled" {
+		c.Uchiwa.Audit.Level = "default"
+	}
+	if c.Uchiwa.Audit.Logfile == "" {
+		c.Uchiwa.Audit.Logfile = "/var/log/sensu-enterprise/sensu-enterprise-dashboard-audit.log"
```
Comment (calebhailey) [inline uchiwa/config/config.go]: Let's update this logfile location to `/var/log/sensu/sensu-enterprise-dashboard-audit.log`

File: uchiwa/helpers.go [hunk 2]
```diff
@@ -39,6 +41,15 @@ func findModel(id string, dc string, checks []interface{}) map[string]interface{
 	return nil
 }
 
+// GetIP returns the real user IP address
+func GetIP(r *http.Request) string {
+	if xForwaredFor := r.Header.Get("X-FORWARDED-FOR"); len(xForwaredFor) > 0 {
```
Comment (amdprophet) [inline uchiwa/helpers.go]: Will this work if Uchiwa isn't running behind a proxy?

Comment (amdprophet) [inline uchiwa/helpers.go]: Also a typo - variable should be named xForwardedFor.

Comment (palourde) [inline uchiwa/helpers.go]: Good catch, I'll correct the typo. Concerning the `X-FORWARDED-FOR` header, if it isn't found (`len(xForwardedFor) >  0`), then we get the `r.RemoteAddr` IP address on line 49.

**Pull request merged**